net/shorewall-core/Makefile

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (C) 2008-2017 OpenWrt.org
```

Comment:
The packages were part of BB. So the Makefiles are copied from BB.
The Copyright for OpenWrt will be the same. I added mine for 2017.

Hope to hear from you.